src/lib.rs

```diff
@@ -58,18 +58,18 @@ extern crate fnv;
 extern crate mopa;
 #[cfg(not(target_os = "emscripten"))]
 extern crate pulse;
+extern crate rayon;
 #[cfg(not(target_os = "emscripten"))]
 extern crate rayon_core;
```

Comment:
Given that `rayon_core` is stable and `rayon` is not, we can just use `rayon_core` where possible I guess.